SUPPLIMENTARY-INFO.md

```diff
@@ -0,0 +1,42 @@
+# Supplimentary Information
+
+## Front End Folder Structure
+
+`/.storybook` Config for storybook.
+
+`/config` Webpack config.
+
+`/cypress`
+
+`/data` Example Runner JSON schemas.
+
+`/public` Public static assets.
+
+`/scripts` NPM scripts for running the app.
+
+`/src` JavaScript source files.
+
+`/src/actions` Redux action creators.
+
+`/src/components` React components.
+
+`/src/constants` Constants that can be used throughout the application such as theme colours and action names.
+
+`/src/containers` Redux container components.
```

Comment:
This directory doesn't exist any more so we should probably double check. Maybe pushing this down into a README.md in each folder is a better idea so they are closer to the changes so they hopefully won't get as out of sync as they have done.

Reply:
Discussed this with Sam to do that at a future time but for now to fix list and leave in place as a reference.

Reply:
Have removed the invalid references, we will create readmes for the directories as required and drop this list but keep it as a reference for now.